roles/confluent.common/molecule/custom-vars/molecule.yml

```diff
@@ -0,0 +1,24 @@
+---
+driver:
+  name: docker
+platforms:
+  - name: custom
+    image: geerlingguy/docker-centos7-ansible
+    command: ""
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    pre_build_image: true
+provisioner:
+  name: ansible
+  playbooks:
+    converge: ../converge.yml
+  inventory:
+    group_vars:
+      all:
+        jmxexporter_enabled: true
+        jolokia_enabled: false
+        custom_yum_repofile: true
+        custom_yum_repofile_filepath: custom-vars/files/custom.repo
```

Comment:
Anything against naming `confluent.repo`?

Reply:
file name/ its contents dont matter, test is that a .repo file gets placed on host